src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -171,45 +179,40 @@ export function AbundanceConfiguration(props: ComputationConfigProps) {
   }, [collectionVarItems, collectionVariable]);
 
   return (
-    <div
-      style={{
-        display: 'flex',
-        gap: '0 2em',
-        padding: '1em 0',
-        alignItems: 'center',
+    <ComputationStepContainer
+      computationStepInfo={{
+        stepNumber: 1,
+        stepTitle: `Configure ${computationAppOverview.displayName} data`,
```

Comment:
```suggestion
        stepTitle: `Configure ${computationAppOverview.displayName}`,
```

Another recent minor wording change for all apps :)